### DISCUSSION TITLE
clipboard-yank, clipboard-paste-* does not work

### DISCUSSION THREAD

**Discussion by RGBCube** [helix-editor/helix#7141] [Q&A] [answered]
### Summary

When using the clipboard-yank and clipboard-paste-* commands, they never use the clipboard.

clipboard-yank does nothing and clipboard-paste-replace replaces the selection with an empty string ().

### Reproduction Steps

I have installed Helix from nixpkgs-unstable, my config is as so:

```nix
{ home-manager.users.nixos = {
  programs.nushell = {
    environmentVariables = {
      EDITOR = "hx";
    };
  };

  programs.helix = enabled {
    settings.theme = "catppuccin_mocha";

    settings.editor = {
      color-modes = true;
      cursor-shape.normal = "bar";
      cursorline = true;
      file-picker.hidden = false;
      line-number = "relative";
      shell = [
        "nu"
        "-c"
      ];
      text-width = 100;
      whitespace.render.tab = "all";
      whitespace.characters.tab = "→";
    };

    settings.editor.auto-pairs = {
      "(" = ")";
      "{" = "}";
      "[" = "]";
      "\"" = "\"";
      "'" = "'";
      "<" = ">";
      "`" = "`";
    };
  };
}; }
```

### Helix log

Noting gets logged when i run these commands.

### Platform

Linux (NixOS)

### Terminal Emulator

Konsole 23.04.1

### Helix Version

helix 23.05

**Comment by kirawi** [comment, marked as answer]
Did you check https://github.com/helix-editor/helix/wiki/Troubleshooting#copypaste-fromto-system-clipboard-not-working?

**Comment by RGBCube** [comment]
> Did you check https://github.com/helix-editor/helix/wiki/Troubleshooting#copypaste-fromto-system-clipboard-not-working?

I would have expected an error to tell me to install these packages

**Reply by levtoji** [reply]
Yes, it says "joined and janked to system clipboard" even though it did not.

**Comment by fschn90** [comment]
Same issue here!
